ad/internal/config/config.go

```diff
@@ -247,12 +255,104 @@ func (c *KerberosTransporter) Post(_ *winrm.Client, request *soap.SoapMessage) (
 	defer resp.Body.Close()
 
 	if resp.StatusCode != 200 {
-		return "", fmt.Errorf("http error while making kerberos authenticated winRM request: %s", err)
+		return "", fmt.Errorf("http error while making kerberos authenticated winRM request: %d - %s ", resp.StatusCode, resp.Status)
```

Comment:
including response body in error could be helpful additional context?

Reply:
Good call.